var/spack/repos/builtin/packages/gatk/package.py

```diff
@@ -39,17 +39,26 @@ class Gatk(Package):
 
     version('3.8-0', '0581308d2a25f10d11d3dfd0d6e4d28e', extension='tar.gz',
             url="https://software.broadinstitute.org/gatk/download/auth?package=GATK")
+    version('4.0.4.0', '083d655883fb251e837eb2458141fc2b',
+            url="https://github.com/broadinstitute/gatk/releases/download/4.0.4.0/gatk-4.0.4.0.zip")
 
     depends_on('java@8:', type='run')
+    depends_on('python@2.6:2.8,3.6:3.7', type='run', when='@4.0:')
```

Comment:
Where did you find these version requirements? I've just never seen something that required exactly Python 3.6 before.

Reply:
GATK 4 requires "Python 2.6 or greater" or "Python 3.6.2". https://github.com/broadinstitute/gatk Did I specify in the correct way?